Source/WebKit/UIProcess/Automation/protocol/BidiBrowser.json

```diff
@@ -1,5 +1,6 @@
 {
     "domain": "BidiBrowser",
+    "exposedAs": "browser",
```

Comment:
wait why not just have `"domain": "browser"`?

Reply:
It is desired to have generated class names to match the filenames because it keeps the Bidi prefix around. Otherwise there is a clash between Inspector's `BrowserBackendDispatcherHandler` and WebDriver's `BrowserBackendDispatcherHandler`.

A lesser reason is that I enjoy having an easy way to grep the generated code for anything WebDriver BiDi related.